src/plugins/dna_export/src/ExportSequenceTask.cpp

```diff
@@ -553,18 +553,31 @@ U2Sequence ExportAnnotationSequenceSubTask::importAnnotatedSeq2Dbi(const SharedA
     importer.startSequence(os, resultDbiRef, U2ObjectDbi::ROOT_FOLDER, ad->name, false);
     CHECK_OP(os, U2Sequence());
 
-    foreach (const U2Region &annotatedRegion, ad->location->regions) {
+    bool isReverseComplement = ad->getStrand().isCompementary() && nullptr != ei.complTT;
```

Comment:
1. rename to "isReverseComplementAndTranslate"
2. do not use Yoda style

Reply:
1. Well, actually, "reverse-complement" and "translate" - two different options. You can "reverse-complement", but not "translate" and vice-versa.
2. OK